testing-standard.md

```diff
@@ -1,57 +1,112 @@
 # HX Testing Standards
 
-## Selenuim Tests
+# 1.0 Unit Tests
 
-TODO
+## 1.1 Our Technology
+In order to keep test writing familiar across the Holiday Extras group, we should use the following tech for writing our unit tests:
 
-## System Tests
+* [Mocha](https://mochajs.org/): Our test framework of choice
+* [Sinon](http://sinonjs.org/): For our stubs & spies
+* [Chai](http://chaijs.com/): For making assertions in our tests
+  * [Sinon-chai](https://github.com/domenic/sinon-chai): For integrating Sinon with Chai
+  * [chai-as-promised](https://github.com/domenic/chai-as-promised/): For clean promise assertions
+* [React Test Utilities](https://facebook.github.io/react/docs/test-utils.html): Helpers for testing React components 
 
-### Each test should:
-* Use product codes that are guaranteed to be available
-* Use dynamic dates that are a fixed amount of time in the future
-* Not assert against any dynamic data (eg. content) that may change unexpectedly
+## 1.2 Good Test Folders
+When addind new tests to projects, test files should sit in a folder hierarchy that mimics the hierarchy fo the source code. The test files themselves should be named similarly to the source files, but with `test` at the beginning of the file name. For example, if we have a source file `src/views/checkoutView.js` we should have a test file `test/unit/views/testCheckoutView.js` or similar. This makes finding tests an easy and logical process.
```

Comment:
I'd vote for not adding `test` to a filename, we and test runners know its a test due to being in the `test` directory.